llmproxy/tests/unit_tests/models/test_vertexai.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+from llmproxy.models.vertexai import VertexAI
+from dotenv import load_dotenv
+
+load_dotenv(".env.test")
+project_id = os.getenv("GOOGLE_PROJECT_ID")
+
+def test_invalid_project_id() -> None:
+    vertexai = VertexAI(project_id="invalid id")
+    response = vertexai.get_completion()
+    assert "Permission denied" in response.message
+
+def test_invalid_location() -> None:
+    vertexai = VertexAI(project_id=project_id, location="test")
+    response = vertexai.get_completion()
+    assert "Unsupported region" in response.message
+
+def test_unsupported_model() -> None:
+    vertexai = VertexAI(project_id=project_id,model="test")
+    response = vertexai.get_completion()
+    assert "Model not supported" in response.message
+
```

Comment:
Also getting 21 warnings about deprecation, we need to make sure this won't affect the codebase in the future

Reply:
looking in to it